client/app/dedicated/server/rendezvous/dedicated-server-rendezvous.routes.js

```diff
@@ -0,0 +1,11 @@
+angular.module('App').config(($stateProvider) => {
+  $stateProvider.state('app.dedicated.server.rendezvous', {
+    url: '/rendezvous',
+    templateUrl: 'dedicated/server/rendezvous/dedicated-server-rendezvous.html',
```

Comment:
Could be imported with an `import` statement. In that way if the file is moved or renamed, when we will use webpack to compile the application, an error will be raise.